### ISSUE TITLE
Include testnets into the dAPI Pricing Merkle Tree

### ISSUE THREAD

**Issue by PrenaamD** [api3dao/dapi-management#113]
The current [dAPI pricing merkle tree](https://api3dao.github.io/dapi-management/data/dapi-pricing-merkle-tree-root/current-hash.json) used by the Market FE does not include testnets 

It was decided [here](https://github.com/api3dao/tasks/issues/308#issuecomment-1820908199) that
- Gas prices will be collected for tesnets
- The subscription period will always be one week

**Comment by BTT21000** [comment]
@vponline is collecting testnets gas data and hence Price MT will contain testnets as per current scripts ?

**Comment by vponline** [comment]
> @vponline is collecting testnets gas data and hence Price MT will contain testnets as per current scripts ?

yep testnet gas data is being collected and the subscription period can be set by chain when running the script 👍
